adopt-a-dog/styles/stylesheet.css

```diff
@@ -0,0 +1,280 @@
+* {
+  box-sizing: border-box;
+}
+body {
+  margin: 0;
+  font-family: 'Open Sans', sans-serif;
+}
+h1, h2, h3 {
+  font-family: 'Noto Serif', serif;
+  color: #183c5d;
+}
+a:link {
+  text-decoration: none;
+}
+a:visited {
+  text-decoration: none;
+}
+a:hover {
+  text-decoration: underline;
+}
+a:active {
+  text-decoration: underline;
+}
+/*---------------------------Header, Main Nav and Footer------------------------------ */
+header {
+  background-image: url("../images/christopher-harris-57366-unsplash.jpg");
+  background-color: white;
+  height: 400px;
+  background-position: center;
+  background-repeat: no-repeat;
+  background-size: cover;
+}
+.header-flex-container {
+  display: flex;
+  flex-wrap: wrap;
+  justify-content: space-between;
+  max-width: 1200px;
+  margin: 0 auto;
+  padding-top: 3%;
+}
+.header-logo img {
+  width: 25rem;
+}
+.header-logo p {
+  color: #8db553;
+}
+.header-cart img {
+  width: 30px;
+  height: auto;
+}
+.header-cart p {
+  color: white;
+}
+.header-cart button {
+  background-color: #8db553;
+  color: white;
+  border-radius: 2px;
+  border-color: #8db553;
+  width: 150px;
+  height: 25px;
+}
+nav.main-nav {
+  background-color: #ececec;
+}
+.main-nav-container {
+  margin: 0 auto;
+  max-width: 1000px;
+  padding: 2% 0;
+}
+.main-nav li {
+  color: #183c5d;
+  font-size: 1.15rem;
+  font-weight: Bold;
+}
+.main-nav ul {
+  display: flex;
+  flex-wrap: wrap;
+  list-style-type: none;
+  margin: 0;
+  padding: 0;
+}
+.main-nav a:visited {
+  color: #183c5d;
+}
+.nav-with-slash::after { /* Adds a / after all nav links except last link */
```

Comment:
Nice! This could also be done using n-th child, so you wouldn't have to maintain your HTML